src/khoj/routers/api_chat.py

```diff
@@ -595,6 +586,16 @@ async def send_rate_limit_message(message: str):
         if ConversationCommand.Notes in conversation_commands and is_none_or_empty(compiled_references):
             conversation_commands.remove(ConversationCommand.Notes)
 
+        if conversation_commands == [ConversationCommand.Help]:
+            conversation_config = await ConversationAdapters.aget_user_conversation_config(user)
+            if conversation_config == None:
+                conversation_config = await ConversationAdapters.aget_default_conversation_config()
+            model_type = conversation_config.model_type
+            formatted_help = help_message.format(model=model_type, version=state.khoj_version, device=get_device())
+            # Adding specification to search online specifically on khoj.dev pages.
+            defiltered_query += " site:khoj.dev"
+            conversation_commands.append(ConversationCommand.Online)
```

Comment:
This is good! There's a slight variation in how I think we should handle this.
1. Rather than explicitly adding the qualifier `site:khoj.dev` to the query here, I think it would be better if we exposed a parameter in the `search_online` function for explicit filters. This will generalize better as we add additional commands and make the code more readable. 
2. I also think we should move this logic back to the top of the function in the event that the user has just run `/help` and nothing else. We want to exit early and save on compute costs if that's the case.